src/js/components/FileInput/FileInput.js

```diff
@@ -247,6 +248,7 @@ const FileInput = forwardRef(
                 icon={<RemoveIcon />}
                 hoverIndicator
                 onClick={(event) => {
+                  if (onClickRemove) onClickRemove();
```

Comment:
WDUT passing the file to `onClickRemove` function? So the callback can decides according with the file

Reply:
Each `onRemove` would be specific to an individual file. However, I could see there being value to having access to that file data.

To be consistent with other handlers I think it would be appropriate to call the onRemove with a signature of something like:
```
const adjustedEvent = event;
adjustedEvent.file = file;
onRemove(adjustedEvent)
```